### ISSUE TITLE
Unable to set hotkeys to space bar

### ISSUE THREAD

**Issue by TarjinderSingh** [tadashi-aikawa/obsidian-various-complements-plugin#305]
I’m encountering an issue while trying to assign the spacebar as the “select” hotkey in the settings of the Various Complements plugin. Despite trying multiple keycode combinations for the spacebar on a Mac, the plugin doesn’t recognize the input. The spacebar is not being accepted as a valid input for the “select” hotkey, despite trying different keycodes.

	•	Is assigning the spacebar as a hotkey supported by the plugin?
	•	If yes, what is the correct keycode for the spacebar on a Mac?

**Comment by tadashi-aikawa** [comment]
@TarjinderSingh
I do not have a Mac environment, so I cannot check the spacebar. However, the value that should be input is the string displayed in 'event.key' when the spacebar is pressed on the website https://www.toptal.com/developers/keycode. If nothing is displayed or there is no response, it is likely that it is not supported.

 
![image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/9500018/f742ceca-77ba-468a-9788-35192fef6214)

**Comment by TarjinderSingh** [comment]
@tadashi-aikawa Thank you for your prompt reply! For the space bar, I see this (blank space) code. I tried putting it into the Settings, but it doesn't work. I also tried event.code and the other things in event.dump, and it didn't seem to work. I guess it's currently not supported then?

<img width="1334" alt="image" src="https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/6049509/98532607-7f5e-40da-b08d-207244a930cb">
